src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs

```diff
@@ -796,6 +821,7 @@ public static void SetCommandsFromFile(string path, CommandType type)
         /// 2) This exception can be due to operation failed.
         /// </exception>
         /// <exception cref="ArgumentException">This exception can be due to an invalid parameter.</exception>
```

Comment:
Impossible AgrumentException on this method

Reply:
Fixed.